File: alphatimer.api.protocol/src/main/java/de/dennisfabri/alphatimer/api/protocol/events/dropped/UnstructuredInputDroppedEvent.java
package de.dennisfabri.alphatimer.api.protocol.events.dropped;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.ToString;

@AllArgsConstructor
@EqualsAndHashCode
@ToString
public class UnstructuredInputDroppedEvent implements InputDataDroppedEvent {

    private final byte[] data;
}

File: protocol-api/src/main/java/de/dennisfabri/alphatimer/api/protocol/events/dropped/UnstructuredInputDroppedEvent.kt
package de.dennisfabri.alphatimer.api.protocol.events.dropped

import de.dennisfabri.alphatimer.api.protocol.data.Bytes

data class UnstructuredInputDroppedEvent(private val data: Bytes) : InputDataDroppedEvent {
    constructor(data: ByteArray) : this(Bytes(data))
}
